### ISSUE TITLE
Tools options doc

### ISSUE THREAD

**Issue by Sleuthgame** [mriale/PyDPainter#121]
Tools bar is documented as a png and PDF but tools options are missing (spacing, filling, etc).

**Comment by Sleuthgame** [comment]
I began writing this. Will be a tools.md and then, fill type and spacing will have their own explanation page.

**Comment by Sleuthgame** [comment]
Hey, Stephane, you seems to have a lot of nice ideas for new functions involving work for the programmer !
But... How's that part of the doc coming along ?
- Heu... Donc voilà... ;-) I'm back on that one.

**Comment by Sleuthgame** [comment]
I have written it up to Text, included. 6 tools left and I'm done.

**Comment by Sleuthgame** [comment]
I add my WIP on the develop branch. The spacing-examples image refuses to display in the spacing.md page.
![Spacing-examples](https://github.com/mriale/PyDPainter/assets/130657737/850aa237-759d-4fe1-ae65-e5b02a0a97a2)

**Comment by mriale** [comment]
The image file name was capitalized, so I capitalized the reference to it. 7e2c9c1dbd384dc2d8077cb685afca970624aea1

I haven't done any other proofreading yet.

**Comment by Sleuthgame** [comment]
Thanks. I'll try to finish the tools list.

**Comment by mriale** [comment]
@Sleuthgame This is amazing all the work you've done on this! Thank you so much!

As you can see, I moved all the tools docs into another directory. That makes some of the the cross-links a little weird but makes the images and such more manageable.

I didn't get to editing fill.md yet. I think there are a few missing images there also. I did add links to the page from the toolbox.md page.

**Comment by Sleuthgame** [comment]
Thank you, that's very kind of you.

I've been struggling to find graphically inspiring examples to give ideas. Some of the functions are a bit dated, which makes sense.

I've also struggled quite a bit with PydPainter's file request between the dev and releases versions. Amiga assigns or favorites are really missing.

I think I found a few bugs along the way too. When you're systematically testing all the options, it's pretty thankless.

Well, the main thing is that we progress at our own pace. I'll have a look at those missing images. I'd like to add a nice example of symmetry.

**Comment by mriale** [comment]
@Sleuthgame Another great explanation -- this time fill types! I edited `fill.md` but noticed that the section about the gradient directions was still not done. I hope I didn't stomp on your edits! I also did some resizing of the images in the markdown to make them a little more consistent.

**Comment by Sleuthgame** [comment]
I DL your .md file and gradient direction snapshots are integrated, but I'm not great at explaining it.
Did the same resizing.

**Comment by Sleuthgame** [comment]
Oh I forgot !  There is a back link to Palette Range to fix because of the new dir.

**Comment by mriale** [comment]
@Sleuthgame Thanks! Yeah, gradient is a little hard to describe. I edited your work to have a few less words, but I hope it still does it justice. I know French is such a descriptive language, and sometimes Americans are too impatient to read any extra words. :smile:

I also fixed the Range link.

I plan to move these updated docs over to production this weekend so that they show up on the PyDPainter web site.
